src/snowflake/cli/api/project/definition_conversion.py

```diff
@@ -383,6 +372,73 @@ def convert_envs_to_v2(pd: ProjectDefinition):
     return None
 
 
+def _convert_templates_in_files(
+    project_root: Path,
+    definition_v1: ProjectDefinition,
+    definition_v2: ProjectDefinitionV2,
+    in_memory: bool,
+):
+    """Converts templates in other files to the new format"""
+    # TODO handle artifacts using the "templates" processor
```

Comment:
what does this mean?

Reply:
Artifacts that use the templates processor to template arbitrary files don't have their templates converted to v2 when using `snow helpers v1-to-v2` (and if the snowflake.yml file doesn't have any templates in it, no warning is shown to the user, so they silently break).

We want to support automatic conversion of these templates eventually, but it's out of scope for this PR, this is just the function where we'd do it so left this comment as a placeholder/hint.